tests/unittests/client/test_experiment_client.py

```diff
@@ -969,3 +974,35 @@ def foo(a, b):
             assert len(params)
             assert "x" in params["a"]
             assert "y" in params["b"]
+
+    def test_parallel_workers(self, monkeypatch):
+        """Test parallel execution with joblib"""
+
+        def foo(x):
+            return [dict(name="result", type="objective", value=x * 2)]
+
+        def optimize(*args, **kwargs):
+            optimize.count += 1
+            return 1
+
+        with OrionState(
+            experiments=[config], trials=generate_trials(base_trial, [], config)
```

Comment:
Why do you pass trials? Could this test be done with no trials?

Reply:
Maybe you should use something like this instead?
```
        with create_experiment(
            exp_config=config, trial_config={}, statuses=[]
        ) as (cfg, experiment, client):
```
So that your client is already created.

Reply:
Did not use this before because need set storage for `experimentclient` explicitly at the beginning for `mulitple-process`, but later I changed to use 'mulitple-threads', this is good enough though.

Reply:
And for the trials, could you pass no trials?

Reply:
There is no trials pass-in now I think.

Reply:
Indeed!